test/common/testvm.py

```diff
@@ -259,7 +273,8 @@ def _kill_ssh_master(self):
             self.ssh_master = None
         if self.ssh_process:
             self.ssh_process.stdin.close()
-            self.ssh_process.wait()
+            if not self._abort:
+                self.ssh_process.wait()
```

Comment:
Yup. the idea is: Lets just kill the ssh master (no change to previous behavior), and let it pick up again if/when needed.